params/params.go

```diff
@@ -0,0 +1,57 @@
+package params
+
+import (
+	"github.com/jfrog/jfrog-client-go/artifactory"
+	"reflect"
+)
+
+type GoResolverDeployer struct {
+	resolver *GoParams
+	deployer *GoParams
+}
+
+func (resolverDeployer *GoResolverDeployer) Resolver() *GoParams {
+	return resolverDeployer.resolver
+}
+
+func (resolverDeployer *GoResolverDeployer) SetResolver(resolver *GoParams) *GoResolverDeployer {
+	resolverDeployer.resolver = resolver
+	return resolverDeployer
+}
+
+func (resolverDeployer *GoResolverDeployer) Deployer() *GoParams {
+	return resolverDeployer.deployer
+}
+
+func (resolverDeployer *GoResolverDeployer) SetDeployer(deployer *GoParams) *GoResolverDeployer {
+	resolverDeployer.deployer = deployer
+	return resolverDeployer
+}
+
+type GoParams struct {
```

Comment:
Suggest renaming *GoParams* to *Params*.

Reply:
Done